assignments/objects.js

```diff
@@ -19,26 +20,79 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
```

Comment:
Good job building out your objects.  You'll work with objects and arrays every day so it is important that you learn them inside and out.